src/airtop/client.py

```diff
@@ -0,0 +1,90 @@
+from .base_client import BaseClient, AsyncBaseClient
+from .wrapper.windows_client import AirtopWindows, AsyncAirtopWindows
+from .wrapper.sessions_client import AirtopSessions, AsyncAirtopSessions
+
+class Airtop(BaseClient):
+    """
+    Use this class to access the different functions within the SDK. You can instantiate any number of clients with different configuration that will propagate to these functions.
+
+    Parameters
+    ----------
+    base_url : typing.Optional[str]
+        The base url to use for requests from the client.
+
+    environment : AirtopEnvironment
+        The environment to use for requests from the client. from .environment import AirtopEnvironment
+
+
+
+        Defaults to AirtopEnvironment.DEFAULT
```

Comment:
How does Python work with defaults?  It seems odd that `environment` is not marked as optional if it has a default.

Reply:
@culli I'm not an expert here, but there's a file called `environment.py` with the following definition.

```
import enum


class AirtopEnvironment(enum.Enum):
    DEFAULT = "https://api.airtop.ai/api/v1"

```

Reply:
As a side comment, the `base_url` prop is optional when creating the client, in which case it will grab the AirtopEnvironment.DEFAULT value.

Reply:
ok perfect!